### ISSUE TITLE
Example Mini game

### ISSUE THREAD

**Issue by Cooldude2606** [explosivegaming/factorio-olympics#64]
**Feature overview**
Name: Example Mini game
TL;DR: Add a fake mini game file which explains in detail what the different events and functions can be used for.

**Details**
Although alot of the functions have ok documention for the params and a description, there isnt much detail of when a function should be used or what the limitations of the functions are. Further more, the events have no documentation add all and currently you are required to read the code where each event is raised in order to understand how it can be used.

**Possible methods**
Added a file that isnt loaded called "example.lua" and add lots of comments to explain the different events and return values.
